<?xml version="1.0"?>

<launch>

    <!-- run pcd_to_linescan node -->
    <node pkg="pcd_to_linescan" type="pcd_to_linescan_node" name="pcd_to_linescan">

        <remap from="cloud_in" to="/PointCloud"/>
        <remap from="scan_out" to="/scan"/>
        <rosparam>
            transform_tolerance: 0.01
            min_height: 0.0
            max_height: 1.0

            angle_min: -1.5708 # -M_PI/2
            angle_max: 1.5708 # M_PI/2
            angle_increment: 0.0087 # M_PI/360.0
            scan_time: 0.3333
            range_min: 0.45
            range_max: 4.0
            use_inf: true
            inf_epsilon: 1.0
        </rosparam>

    </node>

</launch>
